src/sbr/easinTemplates.py

```diff
@@ -40,10 +40,10 @@
 spring:
   profiles: dev
   datasource:
-    url: jdbc:h2:/tmp/{{project.name}}/sbr-gen-database.h2;DB_CLOSE_ON_EXIT=FALSE
+    url: jdbc:postgresql://localhost:5432/essDB
     username: easin
```

Comment:
i will suggest to make username and password of the Database  configurable 
for example replace

`username: easin`
by 
`username :{{project.name}}`.

and 

`password: Easin`
by 
`password:sbr-{{project.name}}-PWD`.

for example in all configurations